layouts/partials/hero.html

```
@@ -0,0 +1,63 @@
+        <div class="hero-home__copy">
+          <div class="hero-home__content">
+            <h1 class="hero-home__title">
+              {{ $titleBefore }}<span class="hero-home__title-accent">{{ $titleHighlight }}</span>{{ $titleAfter }}
+            </h1>
+            {{- with $description }}
+            <p class="hero-home__description">{{ . }}</p>
+            {{- end }}
+            <div class="hero-home__actions-block">
+              <div class="hero-home__actions" role="group" aria-label="Primary actions">
+                {{- if $buttonText }}
+                <a
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)